### PR TITLE
PrivateKey Analyzer

### PR DESCRIPTION
**Pull request by naveen497**

### CODE REVIEW

File: Philips.CodeAnalysis.Common/DiagnosticIds.cs [hunk 1]
```diff
@@ -70,5 +70,6 @@ internal enum DiagnosticIds
 		AvoidAssemblyVersionChange = 2075,
 		AssertFail = 2076,
 		AvoidSwitchStatementsWithNoCases = 2077,
+		AvoidPrivateKeyProperty = 2088
```
Comment (bcollamore) [inline Philips.CodeAnalysis.Common/DiagnosticIds.cs:73]: 2078

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <PackageTags>CSharp MsTest Roslyn CodeAnalysis analyzers Philips</PackageTags>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <DevelopmentDependency>true</DevelopmentDependency>
-    <Version>1.2.1</Version>
+		<Version>1.2.2</Version>
     <AssemblyVersion>1.0.3.0</AssemblyVersion>
     <FileVersion>1.2.1.0</FileVersion>
```
Comment (bcollamore) [inline Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj:33]: Rev the FileVersion to match.  (Not rev'ving assemblyversion is correct.)

**Review by bcollamore — CHANGES_REQUESTED** [review]

File: Philips.CodeAnalysis.Common/DiagnosticIds.cs (hunk 1 above)
Comment (naveen497) [reply Philips.CodeAnalysis.Common/DiagnosticIds.cs:73]: Sorry. Fixed it.

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj (hunk 2 above)
Comment (naveen497) [reply Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj:33]: Done.

**Review by bcollamore — APPROVED** [review]

**Pull request merged**